node/node_test.go

```diff
@@ -40,6 +38,7 @@ import (
 	"github.com/algorand/go-algorand/util"
 	"github.com/algorand/go-algorand/util/db"
 	"github.com/algorand/go-algorand/util/execpool"
+	"github.com/stretchr/testify/require"
```

Comment:
could you please move this back ? our code is always having the vendored code separated.

Reply:
done.